web/themes/joinup/joinup_theme.theme

```diff
@@ -1052,6 +1054,16 @@ function joinup_theme_theme_suggestions_container_alter(array &$suggestions, arr
       $suggestions[] = 'container__edit_actions__search_bar';
     }
   }
+
+  // Early bail out.
```

Comment:
Please remove this, it's not needed, as the check already checks this, and it's not useful to exit function.